config.go

```diff
@@ -0,0 +1,265 @@
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/cheynewallace/tabby"
+	"github.com/urfave/cli/v2"
+	"gopkg.in/yaml.v3"
+)
+
+const (
+	configDir  = "stream-cli"
+	configFile = "config.yml"
+
+	defaultEdgeURL = "https://chat.stream-io-api.com"
```

Comment:
I guess you don't need it since go sdk has it

Reply:
think about custom domains

Reply:
they can provide theirs but our default should be used from go sdk

Reply:
not sure if we export it though

Reply:
ok got it, will figure it out on next iteration when sdk will be involved